models/bert_feature_extractor_model.py

```diff
@@ -55,8 +56,9 @@ def __init__(
 
 
     def forward(self, documents: List[str]) -> Tensor:
-        embeddings = self.compute_embeddings(documents)
-        return self.linear(embeddings)
+        embeddings = self.compute_embeddings(documents) + 0.1
```

Comment:
What is adding 0.1 for?

Reply:
This PR is just whatever temporary state I ended up with yesterday. This 0.1 may be safely removed.